### DISCUSSION TITLE
3060 Ti not being checked?

### DISCUSSION THREAD

**Discussion by Pinija** [jef/streetmerchant#1218] [Help]
The tool seems to work fine on windows, but I haven't got a single "info log" about the rtx 3060 ti status (unlike the 3070 and 3080). Since that is the card I am most interested in, I would like to know if that also means, that there isn't anything being checked for the rtx 3060 ti and how I can fix this?
My parameters are the following:

COUNTRY="germany"
SHOW_ONLY_BRANDS="nvidia" 
STORES="nvidia, alternate, caseking, computeruniverse, cyberport, evga-eu, mediamarkt, nvidia-api, otto, saturn, notebooksbilliger"
When I change the show only series to "3060ti" I get no info messages at all after the parameter print at the start of the bot.
Does anyone know anything about that? 
![No Rtx 3060ti Info](https://user-images.githubusercontent.com/55921663/101283807-26039080-37dd-11eb-95a3-9cd3bab53af4.PNG)

Thanks in advance and thanks to all devs involved in this, really cool tool!!

**Comment by jef** [comment]
It is being checked, bit perhaps not at those stores. If someone contributes then we'll have them!

**Reply by Richierich101** [reply]
I have the same issue, there is no match for rtx3060ti in all of my picked stores, as you can see in the screenshot. Also tried other syntax in, just in case.

![image](https://user-images.githubusercontent.com/16030527/101613979-aa882600-3a0c-11eb-88a3-5405b2668ec8.png)

**Reply by Pinija** [reply]
> I have the same issue, there is no match for rtx3060ti in all of my picked stores, as you can see in the screenshot. Also tried other syntax in, just in case.
> 
> ![image](https://user-images.githubusercontent.com/16030527/101613979-aa882600-3a0c-11eb-88a3-5405b2668ec8.png)

Hey,
As jef said, the stores have to be updated by the items we want to look for. Since I am only interested in the fe edition, which is sold on Notebooksbilliger, i went into the src folder, edited the Notebooksbilliger file by adding a rtx 3060 ti item with the link under which the rtx 3060ti was first sold, while keeping the syntax of the file. If that is the way we are intended to add items, I can make a pull request.

**Reply by Richierich101** [reply]
Ok, makes sense.. That would be great cause I´m looking for a FE edition on NBB, too.
